### PR TITLE
User documentation: Calendar

### PR DESCRIPTION
**Pull request by folaj034**
**Describe the technical changes contained in this PR**
Be clear about purpose and scope of the change

**Previous behaviour**
Describe the product behaviour before this change, if relevant

**New behaviour**
Describe how product behaviour is altered by this change

**Related issues addressed by this PR**
List issue numbers using the "Fixes #xxx" syntax

**Have the following been addressed?**
- [ ] Have test cases been created for all of the changes?
- [ ] Do all of the test cases pass?
- [ ] Has the testing been done using the default docker-compose environment?
- [ ] Are documentation changes required?
- [ ] Does this change break or alter existing behaviour?

### CODE REVIEW

File: docs/Calenda.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#MealPlanner (GoodMealPlan) Documentation
+##Mealplan Calenda
```
Comment (rorybray) [inline docs/Calenda.md]: check spelling

File: docs/Calenda.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+###Overview
+The meal plan Calenda  allows users to view a specific meal plan in the desktop view.
+###Steps to follow:
+1.	Login to GoodMealPlan
+2.	Click on meal plan or  create Meal Plan then Enter the meal plan name 'XXX' 
+3.	Click on the ‘XXX’ mealplan 
+4.	Meal plan name is displayed in tabular form like: 
+
+ ![calenda](https://snipboard.io/eMlWHc.jpg)
```
Comment (rorybray) [inline docs/Calenda.md]: Please checkin images alongside .md files.

**Review by rorybray — CHANGES_REQUESTED** [review]

**Review by rorybray — CHANGES_REQUESTED** [review]
Still need the image links to be updated. They should look like:

      calendar 

and so on.

**Comment by folaj034** [comment]
Hi Rory,
I didn't realize the imagelink  wasn't updated properly. I'll update and
revert.

BR

**Comment by folaj034** [comment]
Hi Rory,
I updated the image link. Pls confirm.

Regards

**Pull request merged**